### PR TITLE
feat(Farms): Sort filter add stable option

### PR DESCRIPTION
**Pull request by Woody810**

### CODE REVIEW

File: apps/aptos/components/Farms/components/Farms.tsx [hunk 1]
```diff
@@ -348,6 +350,10 @@ const Farms: React.FC<React.PropsWithChildren> = ({ children }) => {
                     label: t('Latest'),
                     value: 'latest',
                   },
+                  {
+                    label: t('Stable'),
```
Comment (ChefMomota) [inline apps/aptos/components/Farms/components/Farms.tsx:354]: Rename to `Stableswap`

File: apps/web/src/views/Farms/Farms.tsx [hunk 2]
```diff
@@ -420,6 +422,10 @@ const Farms: React.FC<React.PropsWithChildren> = ({ children }) => {
                     label: t('Latest'),
                     value: 'latest',
                   },
+                  {
+                    label: t('Stable'),
```
Comment (ChefMomota) [inline apps/web/src/views/Farms/Farms.tsx:426]: Rename to `Stableswap`

File: apps/aptos/components/Farms/components/Farms.tsx [hunk 3]
```diff
@@ -348,6 +350,10 @@ const Farms: React.FC<React.PropsWithChildren> = ({ children }) => {
...
+                    value: 'stable',
+                  },
```
Comment (ChefMomota) [inline apps/aptos/components/Farms/components/Farms.tsx:356]: On Aptos side, we can hide it first.

File: apps/web/src/views/Farms/Farms.tsx (hunk 2 above)
Comment (Woody810) [reply apps/web/src/views/Farms/Farms.tsx]: Done.

File: apps/aptos/components/Farms/components/Farms.tsx (hunk 3 above)
Comment (Woody810) [reply apps/aptos/components/Farms/components/Farms.tsx]: Done.

**Comment by ChefMomota** [comment]
@Woody810  You changed wrong file

**Comment by chefilip** [comment]
I think we could add a Stable toggle to show only Stable farms instead of using sort. Using sort stable, it shows no-stable farms as well.

**Comment by ChefMomota** [comment]
But for toggles, it looks like we don't have enough space for another toggle UI.

**Comment by Woody810** [comment]
> @Woody810 You changed wrong file

oops...

**Comment by chefilip** [comment]
One idea is Moving toggles into a popup like settings so we could add more toggles. We should seek for designer'a consult about this before merging the PR

**Pull request closed without merging**